code/modules/mod/mod_theme.dm

```diff
@@ -593,7 +592,7 @@
 		However, the systems used in these suits are more than a few years out of date, \
 		leading to an overall lower capacity for modules."
 	default_skin = "security"
-	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 25, BIO = 100, FIRE = 75, ACID = 75, WOUND = 20)
+	armor = list(MELEE = 15, BULLET = 15, LASER = 10, ENERGY = 10, BOMB = 25, BIO = 100, FIRE = 75, ACID = 75, WOUND = 15)
```

Comment:
this suit is printable by robotics currently itll change with the sec suit rework but keep that in mind